src/System.Management.Automation/utils/PsUtils.cs

```diff
@@ -579,15 +190,6 @@ internal static bool IsRunningOnProcessorArchitectureARM()
             {
                 return false;
             }
-#else
-            // Important:
-            // this function has a clone in Workflow.ServiceCore in admin\monad\src\m3p\product\ServiceCore\WorkflowCore\WorkflowRuntimeCompilation.cs
-            // if you are making any changes specific to this function then update the clone as well.
-
-            var sysInfo = new NativeMethods.SYSTEM_INFO();
-            NativeMethods.GetSystemInfo(ref sysInfo);
```

Comment:
Then the `required processor architecture test` is broken on Unix platforms as we are depending on a win32 API to get process architecture. Can you please open an issue for this?